flyway/migrations/V17__TableValuesFunction.sql

```diff
@@ -0,0 +1,13 @@
+CREATE FUNCTION GetSitterApplicationByStatus
+(
+    @status VARCHAR(225)
```

Comment:
No @  here.

Also, status is a predefined variable in MySQL, please change to something else.